athena/llm_core/llm_core/loaders/llm_config_loader.py

```diff
@@ -0,0 +1,64 @@
+from llm_core.models.llm_config import LLMConfig, LLMConfigModel, RawLLMConfig
+from llm_core.utils.model_factory import create_config_for_model
+import yaml
+from pathlib import Path
+from typing import Dict, Optional
+
+_state: Dict[str, Optional[LLMConfig]] = {"llm_config": None}
+
+def _load_raw_llm_config(path: Optional[str] = None) -> RawLLMConfig:
+    """
+    Loads the LLM configuration from a YAML file and returns a validated LLMConfig.
+    Raises pydantic.ValidationError if something is incorrect in the YAML.
+    """
+    # By default, we assume 'llm_config.yml' is in the same directory
+    if path is None:
+        path = "llm_config.yml"
```

Comment:
You could `def _load_raw_llm_config(path: Optional[str] = "llm_config.yml")` instead of the conditional